app/cells/folio/console/react_picker/_image_fields.slim

```diff
@@ -17,22 +17,25 @@
         .folio-console-thumbnail__img-wrap
           = image(fp)
 
+          - edit_data = { file: serialized_file(fp), \
+                          'file-type' => options[:file_type] }
```

Comment:
tady muzes file_type, kdyz to posles jako `data=edit_data`, tak ti to na pomlcky prevede slim

Reply:
Opraveno.